static/css/s2.css

```diff
@@ -2468,6 +2468,10 @@ input.resourcesLink {
   padding-bottom: 12px;
   border-top: 1px solid #EDEDEC;
 }
+.navBlockAuthorPage {
```

Comment:
this class seems to replace `.authorIndex` but it doesn't have all the same styles. Is there a reason?